src/cljs/owlet/components/activity/embed.cljs

```diff
@@ -12,29 +12,60 @@
 
 (def klipse-container-class ".klipse-container")
 
+(defn click-handler [forward?]
+  (let [iframe (js/document.getElementById "klipseSlides")]
+    (if forward?
+      (.postMessage (.-contentWindow iframe) (js/JSON.stringify #js{:method "next"}) "*")
+      (.postMessage (.-contentWindow iframe) (js/JSON.stringify #js{:method "prev"}) "*")))
+  (when-not @valid?
+    (let [next-button (js/document.getElementById "next-slide")]
+      (when (and (.contains (.-classList next-button) "inactive")
+                 forward?)
+        (.add (.-classList next-button) "animate")
+        (js/setTimeout #(.remove (.-classList next-button) "animate") "820")))))
+
+(add-watch valid? :is-valid
+  (fn [key atom old-state new-state]
+    (let [next-button (js/document.getElementById "next-slide")]
+      (if new-state
+        (do
+          (.remove (.-classList next-button) "inactive")
```

Comment:
since these operations are all in a row and they have the same target expression `(.-classList next-button)`
we can probably refactor it to something like this using the thread first macro
```
(-> (.-classList next-button)
       (.remove  "inactive")
       (.add  "active")
       (.add  "animate"))
```
^ that should also eliminate the need for the `(do ...)` as well.

Reply:
you can also apply the same thing to the else below :)